### PR TITLE
Add MaxSessions parameter to sshd_config.

### PR DESCRIPTION
**Pull request by dgo-**

### CODE REVIEW

File: sshd_config.jinja [hunk 1]
```diff
@@ -23,6 +23,10 @@ PermitRootLogin no
 ClientAliveInterval {{ client_alive_interval }}
 {%- endif %}
 
+{%- if max_sessions %}
+MaxSessions {{ max_startups }}
```
Comment (chr4) [inline sshd_config.jinja]: Is it intentional to use `max_startups` here?

Comment (dgo-) [reply sshd_config.jinja]: fixed

Comment (chr4) [reply sshd_config.jinja]: You need to add it to init.sls, too

Comment (dgo-) [reply sshd_config.jinja]: added

**Review by chr4 — APPROVED** [review]

**Pull request merged**